### PR TITLE
docs need type assertion on proto.Message

### PR DESCRIPTION
**Pull request by zachgersh**
makes a bit more sense with the type assertion and not using the `proto.Message` directly.

### CODE REVIEW

File: docs/_docs/customizingyourgateway.md [hunk 1]
```diff
@@ -104,8 +104,13 @@ Or you might want to mutate the response messages to be returned.
 1. Write a filter function.
    ```go
    func myFilter(ctx context.Context, w http.ResponseWriter, resp proto.Message) error {
-   	w.Header().Set("X-My-Tracking-Token", resp.Token)
-   	resp.Token = ""
+        t, ok := resp.(*external.Tokenizer)
```
Comment (johanbrandhorst) [inline docs/_docs/customizingyourgateway.md]: Maybe even something like `externalpb` to make it more clear what general type this is.
```suggestion
        t, ok := resp.(*externalpb.Tokenizer)
```

Comment (zachgersh) [reply docs/_docs/customizingyourgateway.md]: @johanbrandhorst - yeah I like this. I wondered if we should place the type right above the filter?

**Comment by johanbrandhorst** [comment]
Fuzzit failure is fixed on master, I'll merge this.

**Comment by johanbrandhorst** [comment]
Thanks for the contribution!

**Pull request merged**